packages/plugin-phone/src/phone.js

```diff
@@ -70,6 +71,18 @@ const Phone = SparkPlugin.extend({
   namespace: `phone`,
 
   /**
+   * Indicates if the current browser appears to support webrtc calling. Note:
+   * at this time, there's no way to determine if the current browser supports
+   * h264 without asking for camera permissions
+   * @returns {Promise<Boolean>}
+   */
+  isCallingSupported() {
+    return new Promise((resolve) => {
+      resolve(DetectRTC.isWebRTCSupported);
```

Comment:
I like that it is a promise, allows us to change the method behind the scenes without worrying about implementation.